frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -176,7 +186,16 @@
     </div>
 
     <div class="divider" />
-
+    {#if isAdmin(user)}
+    <div class="space-y-2">
+      <p class="text-2xl mb-4">
+        <span class="link" target="_blank" on:click={() => resetProject()}>
+          {$t('project_page.reset_project_model.title', {name: project?.name})}
+          <span class="i-mdi-open-in-new align-middle" />
```

Comment:
I think this icon should only be used for opening links in new tabs, not opening modals. So far we always use buttons for opening modals, so I think that is what you should do here as well.

Reply:
Good idea. Done.